bindgen/package.json

```diff
@@ -35,6 +35,10 @@
     "assembly/index.ts"
   ],
   "devDependencies": {
-    "near-sdk-core": "^3.2.1"
+    "near-sdk-core": "^3.2.0"
```

Comment:
reverting to an older version of sdk-core?

Reply:
```suggestion
    "near-sdk-core": "^3.2.1"
```